### PR TITLE
Commercial support subpage, contact form, footer edits, navbar edits,…

### PR DESCRIPTION
**Pull request by klijakub**
Hello,
we've added:

a new subpage - /commercial support
new subpage - /contact
misc edits like navbar & footer link, 'new' badge etc
The contact form is linked to the new formbucket. We've created separate buckets for terragrunt and terratest.

### CODE REVIEW

File: docs/assets/js/contact-form.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      plan.prop('checked', true);
+    }
+  }
+
+  function submitForm(e) {
+    e.preventDefault();
+
+    if(validateForm()) {
+      var data = form.serialize(form.get(0), { hash: true });
+      submitToFormSpree(data)
```
Comment (brikis98) [inline docs/assets/js/contact-form.js:27]:
```suggestion
      submitToFormSpree(data);
```

File: docs/assets/js/contact-form.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    }
+
+    function submitToFormSpree(data) {
+      submitButton.html("Sending...");
+      submitButton.prop("disabled", true);
+
+      var postParams = {
+        url: "https://api.formbucket.com/f/buk_7GKMDWyXtHYRco5J0JQgFgan",
```
Comment (brikis98) [inline docs/assets/js/contact-form.js:35]: Shouldn't the JS read this from the HTML? Instead of duplicating it here?

**Review by brikis98 — COMMENTED** [review]
Thanks for the PR!

One design question: it seems like the nav for Terragrunt is quite busy now, and it's hard to tell where one nav item ends and another start. Any way to improve that? Perhaps more spacing or a divider between the nav items?

Screen Shot 2020-10-26 at 10 53 44 AM

Also, could you pull in the latest from `master`?

File: docs/_pages/contact/_contact-form.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
+<div class="contact-form-container">
+  <img class='contact-form-back' src='{{ site.baseurl}}/assets/img/contact/right.svg' alt='contact-form-back' />
+  <div id="error-message"></div>
+  <form id="contact-form" method="post" action="https://formbucket.com/f/buk_7GKMDWyXtHYRco5J0JQgFgan">
```
[the remark this answers is not in the document]
Comment (klijakub) [reply docs/_pages/contact/_contact-form.html:4]: Yeah, we created a separate formbucket for Terratest and Terragrunt
image

**Comment by klijakub** [comment]
> Thanks for the PR!
> 
> One design question: it seems like the nav for Terragrunt is quite busy now, and it's hard to tell where one nav item ends and another start. Any way to improve that? Perhaps more spacing or a divider between the nav items?
> 
> Screen Shot 2020-10-26 at 10 53 44 AM
> 

About the design question, we increased spacing between nav items, our designer has noticed that divider would make it even busier :) 
image

**Comment by klijakub** [comment]
@brikis98 one more thing about the formbucket and the contact form, we didn't add recaptcha v3 like we did it at gruntwork, but I don't know is it a 'must-have' right know, or maybe you want to check if there will be a lot of spam and then we would add it.

**Comment by brikis98** [comment]
We can punt on captcha for now and add it later if it becomes a problem!

File: docs/_includes/get-access.html [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<div class="get-access-cmp">
+    <span>Get access to a library of over 300,000 lines of infrastructure code, all thoroughly tested with Terragrunt.</span>
+    <a href="https://gruntwork.io" target="_blank" class="btn btn-primary">Explore Gruntwork.io</a>
```
[the remark this answers is not in the document]
Comment (brikis98) [reply docs/_includes/get-access.html]: This still seems to be an issue?

[the remark this answers is not in the document]
Comment (klijakub) [reply docs/_includes/get-access.html]: I think it's the same on zeplin and here: 
image
image

[the remark this answers is not in the document]
Comment (brikis98) [reply docs/_includes/get-access.html]: Weird, I ran it locally and still had the old version. Must be some Git or caching thing.

**Review by brikis98 — APPROVED** [review]
LGTM!

**Comment by brikis98** [comment]
This is now announced! https://blog.gruntwork.io/introducing-commercial-support-for-terragrunt-and-terratest-a550f44fde56

Thanks for putting it together 👍

**Pull request merged**